src/Idris/REPL.idr

```diff
@@ -238,8 +239,10 @@ updateFile update
              | Nothing => pure (DisplayEdit emptyDoc) -- no file, nothing to do
          Right content <- coreLift $ readFile f
                | Left err => throw (FileErr f err)
-         coreLift_ $ writeFile (f ++ "~") content
-         coreLift_ $ writeFile f (unlines (update (forget $ lines content)))
+         Right () <- coreLift $ writeFile (f ++ "~") content
```

Comment:
There's a `Core.Core.writeFile` with this behavior.

Reply:
Thanks.